do/README.md

```diff
@@ -0,0 +1,48 @@
+# Testing and Building Cmake Template using Docker and Buildkite
+
+The scripts in this directory serve several purposes.  They help provide consistency
+between your build/test results locally and the results of CI.  They also provide a quick
+and simple way to build and test your code without needing to be concerned about your
+local development environment.
+
+## WARNING
+
+Running these scripts will modify the contents of your Cmake Template directory.  
+Because the contents of this repository will be volume mounted into a Docker container and your
+code will be built and tested, any changes that you wish to preserve in your `build`
+or `logs` directory should be copied elsewhere prior to running these scripts.
+
+## Using the scripts
+
+To build and test your code changes using Docker, simply run the `do/build` script, which will produce 
+a debian package and build for Release.  The `do/test` script will build for test, with coverage, and
+any test results will be printed to the command line.
+
+## Testing on Buildkite
+
+You can also run the Cmake Template tests on [Buildkite](https://buildkite.com/path-robotics/cmake-template).
```

Comment:
Bad link

Reply:
Dorp.  I completely forgot to edit this file.  Fixed.